Yubico.YubiKey/src/Yubico/YubiKey/YubiKeyDeviceListener.cs

```diff
@@ -44,83 +45,84 @@ public class YubiKeyDeviceListener : IDisposable
         /// <summary>
         /// An instance of a <see cref="YubiKeyDeviceListener"/>.
         /// </summary>
-        public static YubiKeyDeviceListener Instance => _lazyInstance.Value;
+        public static YubiKeyDeviceListener Instance => _lazyInstance ??= new YubiKeyDeviceListener();
 
-        private static readonly Lazy<YubiKeyDeviceListener> _lazyInstance =
-            new Lazy<YubiKeyDeviceListener>(() => new YubiKeyDeviceListener());
+        /// <summary>
+        /// Disposes and closes the singleton instance of <see cref="YubiKeyDeviceListener"/>.
```

Comment:
Summary: Instructs the SDK to stop listening for YubiKey events, effectively causing the SDK to go "dormant". Accessing `Instance` or calling `YubiKeyDevice.FindAll()` will restart the listener.

Remarks: Enumerating YubiKeys is actually done via a cache. As such, this cache must be maintained and kept up-to-date. This is done by starting several listening threads that run in the background. These listen for the relevant OS device arrival and removal events. Normally, these background listeners will run starting with the first enumeration call to the SDK and remain active until the process shuts down (i.e. a lazy singleton). But there are cases where you may not want the overhead of these listeners running all the time. While they do their best to not eat CPU, they can sometimes generate log noise, exceptions, etc. This method allows you to stop these background listeners and reclaim as much resources as possible from the SDK. This will not invalidate any existing `IYubiKeyDevice` instances, however you will not receive any additional events regarding that device. Any subsequent calls to `YubiKeyDevice.FindAll`, `YubiKeyDevice.FindByTransport`, or `YubiKeyDeviceListener.Instance` will restart the listening threads.


Suggested documentation updates describing the impact to the caller rather than what's it's doing under the covers.

Reply:
I used this as a basis, but removed references to "threads", since this listener doesn't use threads anymore, and the HID listener only subscribes to whatever the OS does.